### ISSUE TITLE
Tests are broken with not so old raku binary

### ISSUE THREAD

**Issue by thibaultduponchelle** [Leont/path-finder#8]
Hello :smile: 

I just wanted to report that the very recent version of this module is broken with *not so old* raku binary.
I think [this commit](https://github.com/Leont/path-finder/commit/ecf8c817049afd056bd31c483eff146c70ded872) breaks during tests with raku binary **2012.10** which is not so old :smile: 
```
[Path::Finder] ===SORRY!=== Error while compiling /home/tduponchelle/.zef/store/Path-Finder-0.3.1.tar.gz/Path-Finder-0.3.1/lib/Path/Finder.rakumod (Path::Finder)
[Path::Finder] Coercion 'Bool(Any)' is insufficiently type-like to qualify a variable. 
[Path::Finder] Did you mean 'class'?
[Path::Finder] at /home/tduponchelle/.zef/store/Path-Finder-0.3.1.tar.gz/Path-Finder-0.3.1/lib/Path/Finder.rakumod (Path::Finder):252
[Path::Finder] ------> 		my Bool() $success⏏ = ~$item ~~ $matcher;
[Path::Finder]     expecting any of:
[Path::Finder]         constraint
[Path::Finder] at /home/tduponchelle/.zef/store/Path-Finder-0.3.1.tar.gz/Path-Finder-0.3.1/t/vcs.t:7
```

(I haven't tested with other flavors of raku)

I don't know well raku ecosystem, maybe a minimum raku requirement in META6.json could also help (or make the test code a bit more portable).
Anyway, it is perfectly working with latest raku code so you can close this issue if you want no pb for me, but I'm happy to share this report anyway :)

Thibault

**Comment by Leont** [comment]
I assume you meant 2020.10. That breakage is entirely accidental, and is fixed in 0.3.2

**Comment by thibaultduponchelle** [comment]
Yes **2020.10**

Great ! Thank you a lot ! :+1:
